### PR TITLE
7628 - wrong styles fixed

### PR DESCRIPTION
**Pull request by ivandudardb**
I have installed the fonts and corrected the styles that changed due to the font changes

### CODE REVIEW

File: web/themes/custom/customtheme/components/00-base/00-defaults/_01-variables.scss [hunk 1]
```diff
@@ -5,10 +5,31 @@
...
+}
+
+@font-face {
+  font-family: 'Source Sans Pro';
+  src: url('/themes/custom/customtheme/assets/fonts/SourceSansPro-Black.woff2') format('woff2'); /* регулярний стиль */
+  font-weight: 900;
+  font-style: normal;
+}
+
+/* Додайте інші стилі для інших варіацій шрифту, які ви маєте */
```
Comment (HeorhiiBoliuk) [inline web/themes/custom/customtheme/components/00-base/00-defaults/_01-variables.scss:28]: Comments must be on English

File: web/themes/custom/customtheme/components/00-base/00-defaults/_01-variables.scss [hunk 2]
```diff
@@ -5,10 +5,31 @@
...
 */
+@font-face {
+  font-family: 'Source Sans Pro';
+  src: url('/themes/custom/customtheme/assets/fonts/SourceSansPro-Regular.woff2') format('woff2'); /* регулярний стиль */
+  font-weight: 400;
+  font-style: normal;
+}
+@font-face {
+  font-family: 'Source Sans Pro';
+  src: url('/themes/custom/customtheme/assets/fonts/SourceSansPro-Semibold.woff2') format('woff2'); /* регулярний стиль */
```
Comment (HeorhiiBoliuk) [inline web/themes/custom/customtheme/components/00-base/00-defaults/_01-variables.scss:16]: The same, comment must be on English

File: web/themes/custom/customtheme/components/00-base/00-defaults/_01-variables.scss [hunk 3]
```diff
@@ -5,10 +5,31 @@
  * and breakpoint related vars into
  * components/00-base/base/_breakpoints.css
 */
+@font-face {
+  font-family: 'Source Sans Pro';
+  src: url('/themes/custom/customtheme/assets/fonts/SourceSansPro-Regular.woff2') format('woff2'); /* регулярний стиль */
```
Comment (HeorhiiBoliuk) [inline web/themes/custom/customtheme/components/00-base/00-defaults/_01-variables.scss:10]: The same, comment must be on English

File: web/themes/custom/customtheme/components/00-base/00-defaults/_01-variables.scss [hunk 4]
```diff
@@ -5,10 +5,31 @@
...
+@font-face {
+  font-family: 'Source Sans Pro';
+  src: url('/themes/custom/customtheme/assets/fonts/SourceSansPro-Semibold.woff2') format('woff2'); /* регулярний стиль */
+  font-weight: 700;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'Source Sans Pro';
+  src: url('/themes/custom/customtheme/assets/fonts/SourceSansPro-Black.woff2') format('woff2'); /* регулярний стиль */
```
Comment (HeorhiiBoliuk) [inline web/themes/custom/customtheme/components/00-base/00-defaults/_01-variables.scss:23]: The same, comment must be on English

**Review by HeorhiiBoliuk — CHANGES_REQUESTED** [review]

**Review by HeorhiiBoliuk — APPROVED** [review]
LGTM

**Review by lavrenchuk-dev — APPROVED** [review]

**Pull request merged**